mars/dataframe/sort/psrs.py

```diff
@@ -48,6 +54,23 @@ def __gt__(self, other):
 _largest = _Largest()
 
 
+class _ReversedValue:
+    def __init__(self, value):
+        self._value = value
+
+    def __lt__(self, other):
+        if isinstance(other, _ReversedValue):
```

Comment:
```suggestion
        if type(other) is _ReversedValue:
```
will be faster as there are no inherited types.